src/main/java/managers/DatabaseManager.java

```diff
@@ -0,0 +1,60 @@
+package managers;
+
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+import java.io.FileReader;
+import java.io.IOException;
+
+import objects.Database;
+
+public class DatabaseManager {
+
+    public Database fullDatabase;
+
+    public DatabaseManager() {
```

Comment:
This seems to be for initializing the database - perhaps use a DatabaseFactory to do this instead of having it in the constructor? And instead have a constructor for the database take in a JSONObject argument.